src/main/java/org/mskcc/cmo/messaging/NatsConnection.java

```diff
@@ -0,0 +1,27 @@
+package org.mskcc.cmo.messaging;
+
+import io.nats.streaming.StreamingConnection;
+import io.nats.streaming.StreamingConnectionFactory;
+import java.io.IOException;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class NatsConnection {
```

Comment:
Looks like all references in the docs are to NATS, not Nats - can we adjust the class name to NATSConnection?